src/view/com/post-thread/PostThreadItem.tsx

```diff
@@ -513,25 +522,88 @@ export const PostThreadItem = observer(function PostThreadItem({
           <Link
             style={[
               styles.loadMore,
-              {borderTopColor: pal.colors.border},
-              pal.view,
+              treeView
+                ? {
+                    borderRadius: 8,
+                    marginHorizontal: 8,
+                    marginTop: 4,
+                  }
+                : {
+                    paddingLeft: 80,
+                    borderTopWidth: 1,
+                    borderTopColor: pal.colors.border,
+                  },
+              treeView ? pal.viewLight : pal.view,
             ]}
             href={itemHref}
             title={itemTitle}
             noFeedback>
-            <Text style={pal.link}>Continue thread...</Text>
+            <Text style={treeView ? pal.textLight : pal.link}>
+              Continue thread...
+            </Text>
             <FontAwesomeIcon
               icon="angle-right"
-              style={pal.link as FontAwesomeIconStyle}
+              style={
+                (treeView ? pal.textLight : pal.link) as FontAwesomeIconStyle
+              }
               size={18}
             />
           </Link>
         ) : undefined}
-      </>
+      </PostOuterWrapper>
```

Comment:
Maybe need a border here to separate threads?
<img width="730" alt="Screen Shot 2023-09-19 at 4 35 35 PM" src="https://github.com/bluesky-social/social-app/assets/4732330/b451d2b9-153a-4dc0-a505-2213a67a0bba">

Reply:
Oh nvm, makes sense like this where it shows a sub-tree that doesn't start from the root
<img width="678" alt="Screen Shot 2023-09-19 at 4 36 32 PM" src="https://github.com/bluesky-social/social-app/assets/4732330/bda9d11e-e9c6-4728-921f-62591dbdc4f5">

Reply:
No replies has an extra line
<img width="738" alt="Screen Shot 2023-09-19 at 4 38 02 PM" src="https://github.com/bluesky-social/social-app/assets/4732330/8807b5b9-16d7-4f62-9a0e-4c1c0fe23d4d">